lib/linguist/languages.yml

```
@@ -5567,9 +5567,10 @@ Protocol Buffer Text Format:
   - text proto
   - protobuf text format
   extensions:
-  - ".textproto"
+  - ".txtpb"
   - ".pbt"
   - ".pbtxt"
+  - ".textproto"
```

Change of plan. It looks like `textproto` is still the primary extension:

```suggestion
  - ".textproto"
  - ".pbt"
  - ".pbtxt"
  - ".txtpb"
```

---

I think `.txtpb` ought to be the primary extension, according to the docs: https://protobuf.dev/reference/protobuf/textformat-spec/#text-format-files.